clang-tidy-checks/setup.sh

```diff
@@ -53,6 +53,9 @@ function build() {
         -DLLVM_BUILD_UTILS=OFF \
         -GNinja ../llvm
   cmake --build . --target clang-tidy
+  cp bin/clang-tidy /bin
```

Comment:
This logic should be the `Dockerfile` instead so this still works locally without removing the llvm checkout